packages/nx/src/native/tui/lifecycle.rs

```diff
@@ -282,6 +282,12 @@ impl AppLifeCycle {
 
 #[napi]
 pub fn restore_terminal() -> Result<()> {
+    // Restore the terminal to a clean state
+    if let Ok(mut t) = Tui::new() {
+        if let Err(r) = t.exit() {
+            debug!("Unable to exit Terminal: {:?}", r);
+        }
+    }
```

Comment:
This part feels a bit weird, we are creating a new instance of the tui just to exit it? Surely we could pull the stuff from the exit handler to make this read better.